#2845 Fixed message validation in event search

Patrykb0802 opened the pull request.

Patrykb0802 committed:
#2845 Fixed message validation in event search

Limraj commented on WebContent/WEB-INF/jsp/events.jsp:
```diff
@@ -123,11 +123,25 @@
-        } else if(!isValid(maxResults.value)) {
-            $set("searchMessage", $get("maxResultsLabel") + " - Incorrect input data type");
-        } else {
+        if (!isValid(eventId.value)) {
```
It needs to be done in the same way as it was done, as in the problem in the topic, correct the message itself, this solution is unreadable, we have to calculate the condition in many places, it is unreadable and will generate increasingly worse code.

Patrykb0802 committed:
#2845 Fixed message validation in event search

- Simplified validation for event id and max results checking if these two parameters are positive integers

Patrykb0802 committed:
#2845 Fixed message validation in event search

Patrykb0802 committed:
#2845 Fixed message validation in event search

- Separated search form validation function from search function
- Changed messages display mechanism

Limraj commented on WebContent/WEB-INF/jsp/events.jsp:
```diff
@@ -119,19 +119,30 @@
 
+    function validateSearchForm(){
+      let messages = [];
+      if (!isValid(eventId.value)) {
```
Where do these eventId values ​​come from? 
Just use the same scheme as in the previous task #2919

Patrykb0802 committed:
#2845 Fixed message validation in event search

- Changed validation mechanism

Patrykb0802 committed:
#2845 Fixed message validation in event search

- Changed variables names for more appropriate

Limraj reviewed (approved):
An error message appears below the field with the incorrect value. Messages are reset if the field value is corrected:

![image](https://github.com/SCADA-LTS/Scada-LTS/assets/35842300/904d1b55-4457-4659-8b9b-4d738cc54d70)

![image](https://github.com/SCADA-LTS/Scada-LTS/assets/35842300/85fadaa6-dc60-4899-aed7-3ab63e7da782)

![image](https://github.com/SCADA-LTS/Scada-LTS/assets/35842300/5cbb338f-6e94-4cbf-868d-4203ff35b521)

![image](https://github.com/SCADA-LTS/Scada-LTS/assets/35842300/246e20e8-53e6-4456-95dd-6fc8071c5bfd)
